src/main/java/gameboy/gamepack/racinggame/data/entity/RacingTrack.java

```diff
@@ -3,45 +3,50 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class RacingTrack {
+import gameboy.gamepack.racinggame.data.dto.RaceResultDto;
 
-    private List<Driver> drivers = new ArrayList<>();
+public class RacingTrack {
 
-    public RacingTrack(int driverCount) {
-        validationDriverCount(driverCount);
-        engageDriver(driverCount);
+    private List<Car> cars;
+    private Referee referee;
+    public RacingTrack(List<Car> cars, Referee referee) {
```

Comment:
```suggestion
    private List<Car> cars;
    private Referee referee;
    
    public RacingTrack(List<Car> cars, Referee referee) {
```

멤버변수와 메소드사이는 개행하는 것이 컨벤션입니다.